app/models/report.rb

```diff
@@ -14,6 +16,18 @@ class Report < ActiveRecord::Base
 
   after_commit :send_report_notification, :on => :create
 
+  def item
+    if item_type == POST
+      Post.find_by(id: item_id)
+    elsif item_type == COMMENT
+      Comment.find_by(id: item_id)
+    end
+  end
```

Comment:
I am not quite sure how I can do that.

I could use something like [constantize](http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-constantize) to get rid of `POST` and `COMMENT`